upload/upload.go

```diff
@@ -168,7 +168,8 @@ func uploadFiles(files, outFiles []string, targetDir string, config *Config) err
 
 	for k, filename := range files {
 
-		log.Infof("Uploading %s with config %s", filename, *configPath)
+		log.Infof("Uploading %s with config %s\n", filename, *configPath)
```

Comment:
```suggestion
```
right? see also below

Reply:
Both tests and "log not reaching the user" seem to work, so it should be fine like this